### PR TITLE
release: remove sample web ui

### PR DESCRIPTION
**Pull request by jaysachs**
lein is very finicky, and requires contortions to integrate into the bazel build (things like self-update, etc).

It's currently broken for internally used JDKs. It also only supports JDK8. 

#3600

### CODE REVIEW

**Comment by schroederc** [comment]
`//kythe/release:release_test` needs fixed.

**Comment by creachadair** [comment]
Could you please add some more context to the commit log?

I'm not saying we shouldn't remove—but it would be nice to have rationale in the log. 

If it does work: Why not release it?
If it doesn't work: do we have intent to fix it, or should we also consider deleting it?

**Comment by creachadair** [comment]
Thanks for updating the description! Is there an issue number we can track here too, around fixing and/or replacing Leiningen?

**Comment by jaysachs** [comment]
Just added the issue, and updated the description.

**Comment by creachadair** [comment]
Thank you!

**Comment by jaysachs** [comment]
So I think I can work around the buildbot issues with something like

```
--- a/kythe/release/release_test.sh
+++ b/kythe/release/release_test.sh
@@ -63,7 +63,12 @@ tools/triples < "$TEST_REPOSRCDIR/kythe/testdata/test.entries" >/dev/null
 
 rm -rf "$TMPDIR/java_compilation"
 export KYTHE_OUTPUT_FILE="$TMPDIR/java_compilation/util.kzip"
-export KYTHE_JAVA_RUNTIME_OPTIONS="-Xbootclasspath/a:$JAVA_LANGTOOLS"
+if (java -version |& head -1 | grep 1.8 >/dev/null);
+then
+  export KYTHE_JAVA_RUNTIME_OPTIONS="-Xbootclasspath/p:$JAVA_LANGTOOLS"
+else
+  export KYTHE_JAVA_RUNTIME_OPTIONS="-Xbootclasspath/a:$JAVA_LANGTOOLS"
+fi
 JAVAC_EXTRACTOR_JAR=$PWD/extractors/javac_extractor.jar \
   KYTHE_ROOT_DIRECTORY="$TEST_REPOSRCDIR" \
   KYTHE_EXTRACT_ONLY=1 \

```
but that seems really awful.

File: kythe/release/release_test.sh [hunk 1]
```diff
@@ -63,7 +63,12 @@ tools/triples < "$TEST_REPOSRCDIR/kythe/testdata/test.entries" >/dev/null
 
 rm -rf "$TMPDIR/java_compilation"
 export KYTHE_OUTPUT_FILE="$TMPDIR/java_compilation/util.kzip"
-export KYTHE_JAVA_RUNTIME_OPTIONS="-Xbootclasspath/p:$JAVA_LANGTOOLS"
+if (java -version |& head -1 | grep 1.8 >/dev/null);
+then
+  export KYTHE_JAVA_RUNTIME_OPTIONS="-Xbootclasspath/p:$JAVA_LANGTOOLS"
+else
+  export KYTHE_JAVA_RUNTIME_OPTIONS="-Xbootclasspath/a:$JAVA_LANGTOOLS"
+fi
```
Comment (jaysachs) [inline kythe/release/release_test.sh]: Anyone OK with this hack?

Comment (shahms) [reply kythe/release/release_test.sh]: Anything involved in attempting to support multiple versions of Java across the 8/9 boundary is going to be a hack and this seems acceptable until we can remove support for < 8 entirely.

File: kythe/release/release_test.sh [hunk 2]
```diff
@@ -100,29 +105,3 @@ fi
 
 # Ensure kythe tool is functional
 tools/kythe --api srv nodes 'kythe:?lang=java#pkg.Names'
-
-tools/http_server \
-  --public_resources web/ui \
```
Comment (schroederc) [inline kythe/release/release_test.sh]: Only delete this line and 117.

Comment (jaysachs) [reply kythe/release/release_test.sh]: done

**Pull request merged**